salt/loader.py

```diff
@@ -344,6 +349,32 @@ def pillars(opts, functions, context=None):
     return FilterDictWrapper(ret, '.ext_pillar')
 
 
+def jinja_filters(functions, utils):
+
+    _jinja_filters_obj = {}
+
+    if not HAS_JINJA_UTILS:
+        return _jinja_filters_obj
+
+    for fun_name in functions.keys():
+        fun_obj = functions[fun_name]
+        if salt.utils.jinja.JINJA_FILTER_KEY not in fun_obj.__dict__:
+            continue
+        if fun_obj.__dict__[salt.utils.jinja.JINJA_FILTER_KEY] is not True:
+            # JINJA_FILTER_KEY must be set as key and True as value
+            continue  # otherwise ignore
+        if salt.utils.jinja.JINJA_FILTER_NAME_KEY in fun_obj.__dict__ and \
+           isinstance(fun_obj.__dict__[salt.utils.jinja.JINJA_FILTER_NAME_KEY], (six.string_types, six.text_type)):
+            jinja_filter_name = fun_obj.__dict__[salt.utils.jinja.JINJA_FILTER_NAME_KEY]
+        else:
+            jinja_filter_name = fun_name.replace('.', '_').lower()  # dns.AAAA becomes dns_aaaa
```

Comment:
I'd vote for consistency. If it is `foo.bar` when using `{{ salt['foo.bar']("baz") }}`, then it should be `foo.bar` when using `{{ "baz" | foo.bar }}`.